### DISCUSSION TITLE
Fragment shaders use

### DISCUSSION THREAD

**Discussion by AaronPorts** [haasn/libplacebo#228] [Q&A]
Can I use fragment shaders with libplacebo? Or rewriting them to a .hook syntax is the only option?

**Comment by haasn** [comment]
User shaders are fragment shaders by default, so I don't really understand the question.

**Reply by AaronPorts** [reply]
I meant using frag shaders with GLSL ES 100 syntax for ex. But it seems that transpilation to .hook is not so difficult.
